server/channels/api4/websocket_test.go

```diff
@@ -424,10 +424,14 @@ func TestWebSocketUpgrade(t *testing.T) {
 	th := Setup(t)
 	defer th.TearDown()
 
+	buffer := &mlog.Buffer{}
+	err := mlog.AddWriterTarget(th.TestLogger, buffer, true, mlog.StdAll...)
+	require.NoError(t, err)
+
```

Comment:
Instead of capturing log output all the time into a buffer, just do it manually once here as needed. (Though I'm not convinced this assertion is really needed in the first place...)